cwms-data-api/src/main/java/cwms/cda/api/ForecastInstanceController.java

```diff
@@ -0,0 +1,209 @@
+import io.javalin.plugin.openapi.annotations.OpenApiParam;
+import io.javalin.plugin.openapi.annotations.OpenApiRequestBody;
+import io.javalin.plugin.openapi.annotations.OpenApiResponse;
+import org.jetbrains.annotations.NotNull;
+import org.jooq.DSLContext;
+import java.util.logging.Logger;
+
+import static com.codahale.metrics.MetricRegistry.name;
+import static cwms.cda.api.Controllers.FORECAST_DATE_TIME;
+import static cwms.cda.api.Controllers.GET_ONE;
+import static cwms.cda.api.Controllers.ID_MASK;
+import static cwms.cda.api.Controllers.ISSUE_DATE_TIME;
+import static cwms.cda.api.Controllers.LOCATION;
+import static cwms.cda.api.Controllers.LOCATION_MASK;
+import static cwms.cda.api.Controllers.NOT_SUPPORTED_YET;
+import static cwms.cda.api.Controllers.OFFICE;
+import static cwms.cda.api.Controllers.RESULTS;
+import static cwms.cda.api.Controllers.SIZE;
+import static cwms.cda.api.Controllers.SOURCE_ENTITY;
+import static cwms.cda.api.Controllers.SPEC_ID;
+import static cwms.cda.api.Controllers.STATUS_200;
+import static cwms.cda.api.Controllers.STATUS_400;
+import static cwms.cda.api.Controllers.STATUS_404;
+import static cwms.cda.api.Controllers.STATUS_501;
+
+public class ForecastInstanceController implements CrudHandler {
+    private static final Logger logger = Logger.getLogger(ForecastInstanceController.class.getName());
+
+    public static final String TAG = "Forecast";
+    private final MetricRegistry metrics;
+
+    private final Histogram requestResultSize;
+
+    public ForecastInstanceController(MetricRegistry metrics) {
+        this.metrics = metrics;
+        String className = this.getClass().getName();
+        requestResultSize = this.metrics.histogram((name(className, RESULTS, SIZE)));
+    }
+
+    private Timer.Context markAndTime(String subject) {
+        return Controllers.markAndTime(metrics, getClass().getName(), subject);
+    }
+
+    protected DSLContext getDslContext(Context ctx) {
+        return JooqDao.getDslContext(ctx);
+    }
+
+    @OpenApi(
+            description = "Used to create and save a forecast instance",
+            requestBody = @OpenApiRequestBody(
+                    content = {
+                            @OpenApiContent(from = ForecastInstance.class, type = Formats.JSONV2)
+                    },
+                    required = true
+            ),
+            method = HttpMethod.POST,
+            path = "/forecast-instance",
+            tags = TAG
+    )
+    @Override
+    public void create(@NotNull Context ctx) {
+        try (final Timer.Context ignored = markAndTime(GET_ONE)) {
+            throw new UnsupportedOperationException(NOT_SUPPORTED_YET);
+        }
+    }
+
+    @OpenApi(
+            description = "Used to delete forecast instance data based on unique fields",
+            queryParams = {
+                    @OpenApiParam(name = FORECAST_DATE_TIME, required = true, description = "Specifies the " +
```

Comment:
I believe only the issue date is in the unique identifier, forecast date shouldn't be needed (double check cause I might have that backwards)

Reply:
![image](https://github.com/USACE/cwms-data-api/assets/156228995/9c15cd22-e507-4b23-9de6-3052887369d5)

Reply:
this means it's a part of uniqueness right?

Reply:
done...for now?